provisioning/buffer/kubernetes/templates/etcd.yaml

```diff
@@ -4,9 +4,9 @@
 resources:
   requests:
     cpu: "200m"
-    memory: "768Mi"
+    memory: "2000Mi"
   limits:
-    memory: "768Mi"
+    memory: "2000Mi"
```

Comment:
Všade navrhujem nastavit `2GB` pre buffer-etcd, nevieme odhadnúť aká bude zátaž o týždeň, potrebujeme to sledovať.